lib/std/special/init-exe/src/main.zig

```diff
@@ -1,5 +1,5 @@
 const std = @import("std");
 
 pub fn main() anyerror!void {
-    std.log.info(.app, "All your codebase are belong to us.\n", .{});
+    std.log.info(.app, "All your codebase are belong to us.", .{});
```

Comment:
Using `std.log` here means that the message is only output in debug mode (using the default log level). Might be potentially confusing for Zig newcomers who create their first program, try to build it in any of the release modes, and see no output?

Reply:
With @heidezomp's latest change, this can be:

```suggestion
    std.log.info("All your codebase are belong to us.", .{});
```